lib/controllers/authors.js

```diff
@@ -0,0 +1,22 @@
+const { Router } = require('express');
+const Author = require('../models/Author.js');
+
+module.exports = Router()
+  .get('/', async (req, res) => {
```

Comment:
make sure you're placing ```.get('/')```... after ```.get('/:id`)```... because in some circumstances express will run through the functions and execute the first match, and since ```/:id``` starts with a slash, sometimes it will do the ```'/'``` function when you want it to do the ```/:id``` function